tests/integration/components/bs-modal-test.js

```diff
@@ -502,3 +502,44 @@ test('Renders in place (no wormhole) if renderInPlace is set', function(assert)
   assert.equal(this.$('.modal').length, 1, 'Modal exists.');
   assert.notEqual(this.$('.modal').parent().attr('id'), 'ember-bootstrap-modal-container');
 });
+
+test('Removes "modal-open" class when component is removed from view', function(assert) {
+  this.set('renderComponent', true);
+  this.render(hbs`{{#if renderComponent}}{{#bs-modal title="Simple Dialog"}}Hello world!{{/bs-modal}}{{/if}}<div id="ember-bootstrap-modal-container"></div>`);
+
+  let done = assert.async();
+
+  // wait for fade animation
+  setTimeout(() => {
+    assert.ok($('body').hasClass('modal-open'), 'body element has "modal-open" class.');
+
+    this.set('renderComponent', false);
+
+    // wait for fade animation
+    setTimeout(() => {
```

Comment:
Oh. Interesting. The docs would benefit explaining this distinction. Often I use the `{{#if}}` method because it make more cognitive sense especially if the modal will be re displayed again.

One of the odd things I run into with `open` is a syncing between setting that to false and the user clicking the backdrop and the model is closed. If I'm not careful to adjust the property that `open` is bound to then it won't observe it being set back to `true` later.

However, if I wrap the modal in `{{#if}}` then it makes more sense, to me anyway, to remember to add a `closedAction=(action (mut theOpenBoundProperty) false)`.

Anyway I'll fix the tests.